help_to_heat/templates/frontdoor/schemes.html

```diff
@@ -2,52 +2,186 @@
-                <p class="govuk-body">{{_("If you need to pay anything, you’ll find out after the assessment. You can decide not to go ahead if you do not agree with the assessment or costs.")}}</p>
-                <p class="govuk-body">{{_("The most common measures installed under this scheme are loft insulation or cavity wall insulation. These may require a financial contribution. Other insulation measures such as solid wall insulation may be available, but the energy supplier is only likely to offer them to you with a substantial financial contribution.")}}</p>
-                <p class="govuk-body">{{_("If you are comfortable with this then in the next step you can create a referral that will be sent to your chosen energy supplier.")}}</p>
-                <p class="govuk-body">{{_("By submitting a referral through this service your energy supplier will also check if you are eligible for additional help through the government’s ECO4 scheme. ECO4 is available for households on low incomes, receiving qualifying benefits, or who are vulnerable to the effects of living in a cold home.")}}</p>
-              </p>
+    {% call macros.form_wrapper(prev_url, button_label=_("Continue to create a referral")) -%}
+    {{ csrf_input }}
+
+    <h1 class="govuk-heading-l">{{ _("We think you might be eligible") }}</h1>
+    <p
+      class="govuk-body">{{ _("Based on your answers, the Great British Insulation Scheme might be suitable for you.") }}</p>
+    <div class="govuk-warning-text">
+      <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+      <strong class="govuk-warning-text__text">
+        <span class="govuk-visually-hidden">Warning</span>
+        {{ _("Please read the following details carefully") }}
+      </strong>
+    </div>
+    <p
+      class="govuk-body">{{ _("The Great British Insulation Scheme is delivered by energy suppliers and their installation partners. They will assess your home, either by a desktop assessment or by arranging a visit to your home. They will determine if they can offer you insulation. ") }}</p>
+    <h2 class="govuk-heading-m">{{ _("What you could be eligible for") }}</h2>
+    {% if text_flags["park_home_text"] %}
+      <p class="govuk-body">{{ _("Based on your answers, you may be able to get the following:") }}</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>{{ _("park home insulation") }}</li>
+      </ul>
+      <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title"
+         data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header"></div>
+        <div class="govuk-notification-banner__content">
+          <p class="govuk-notification-banner__heading"></p>
```

Comment:
I need to check, but I believe I've retained these from before. They show as new because I duplicated the existing elements for the Park Home contribution section.

Reply:
As we found, this ensures the box is spaced correctly as with the design